kuma/wiki/templates/wiki/feed_docs_updated.html

```diff
@@ -0,0 +1,12 @@
+<p>
+  <a href="{{ obj.parent.get_absolute_url()|absolutify }}" title="{{ obj.parent.title }}">
+    {% trans locale=obj.parent.locale %}View {locale} parent{% endtrans %}
+  </a>
+  (<a href="{{ mod_url|absolutify }}">{% trans modified=obj.parent.modified %}last modified at {modified}{% endtrans %}</a>)
```

Comment:
Sorry, can we add the `add_utm` helper to all the urls? That way we count every feed-driven session in the feed campaign.